src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/resettlementassessmentstrategies/YamlResettlementAssessmentStrategy.kt

```diff
@@ -30,6 +31,23 @@ import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.repository.
 import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service.getClaimFromJWTToken
 import java.time.LocalDateTime
 
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
```

Comment:
We should be able to delete this - if it's used then that's probably just a mistake and we can change it to be a string instead